mod/wikirate/features/metric_create.feature

```diff
@@ -8,6 +8,6 @@ Feature: create metrics
     When I go to new metric
     And I choose "Researched"
     And I fill in "Metric Title" with "MyResearch"
-    And I fill in "Question" with "my question"
-    And I fill in "Value Type" with "Number"
+    And I fill in "card[subcards][+question][content]" with "my question"
```

Comment:
textarea does not have the id linked to the label so I have to use it's name field.